### DISCUSSION TITLE
Q&A kick-off

### DISCUSSION THREAD

**Discussion by pattacini** [robotology/community#1] [Feedback & Meta]
Q&A Kick-off.
Any idea about the name? Is _QA_ ok?

cc @lornat75 @paulfitz @randaz81 @vtikha

**Comment by lornat75** [comment]
@drdanz do we need a specific team/group to allow people to post issues?

**Comment by alecive** [comment]
@lornat75 as far as I know, any github user can post an issue in any github public repository, so I would say no :)

**Comment by kt10aan** [comment]
I think "Support" is more self explanatory :)

Any github user can post an issue in any github public repository, however, for a user to set a label for that issue, he has to be in the "contributors" group.

**Comment by pattacini** [comment]
the ability of creating new labels should be put under strict control, indeed :)

**Comment by lornat75** [comment]
@alecive, @kt10aan noted in: d639cd58596b3b9f4c24056492b95954ee85c063

**Comment by alecive** [comment]
@pattacini problem is that only the `owners` group can assign a label to an issues. e.g. I cannot assign any label to this issue even though I am both in the `YARP developers` and in the `iCub developers` group.

**Comment by drdanz** [comment]
@lornat75 confirming @alecive reply, any GitHub user can post issues on a public repository

**Comment by pattacini** [comment]
@alecive I think restraining to `owners` is ok indeed :)
Owners = Gang of Four

**Comment by alecive** [comment]
@pattacini it means that you guys have to manually label any new issue because anybody else cannot do that. It can become quite annoying after a while.
My point was that labeling is a powerful process, and it would be nice if issues were already labeled by the users, instead of being labeled by the contributors. But I don't think that we have another choice.

**Comment by pattacini** [comment]
new labels can be proposed and then, once they get accepted, attached to open issues; otherwise, it will soon become too wild.

**Comment by pattacini** [comment]
It seems this post has some information about the _labeling_ issue:
https://groups.google.com/forum/#!topic/tardis-devel/6IQCFJxxROk

**Comment by alecive** [comment]
The quickest way would be to create another group, e.g. `robotology-collaborators`, gradually put everyone in it, and assign full admin access to that group on this repository (i.e. only `robotology/QA`). A bit risky, though it is a manageable risk.

**Comment by pattacini** [comment]
I've created the team `qa-users` with _write-access_ to address the labeling issue; we need to populate it with users...

**Comment by drdanz** [comment]
In case we want to import issues from other repositories to the QA one, this is a nice tool: https://github.com/IQAndreas/github-issues-import
